source/cellular/known-working-3g-4g-modems.rst

```diff
@@ -122,6 +117,8 @@ for help, **do not send us e-mail asking for support or drivers**.
 Modems reported to work as Ethernet devices
 -------------------------------------------
 
+See `here <https://redmine.pfsense.org/issues/6226>`__ for info on how to install and use **usb_modeswitch**.
```

Comment:
Since that issue has not been officially implemented, it involves taking actions that shouldn't be performed on a production firewall, so I would not feature it so prominently.

For future reference: Avoid using "here" links ( https://www.w3.org/QA/Tips/noClickHere )

`**usb_modeswitch**` should be ` ``usb_modeswitch`` `